targets/targets.json

```diff
@@ -4686,7 +4686,8 @@
             "CMSIS_NVIC_VIRTUAL",
             "LPTICKER_DELAY_TICKS=3",
             "MBED_MPU_CUSTOM",
-            "BL2"
+            "BL2",
```

Comment:
Do we need `#define BL2` for Musca S1 in `region_defs.h` similar to https://github.com/ARMmbed/mbed-os/pull/14187/commits/9c31de2eafe738454ff4a8ee29cac374374caaf3#diff-f120407ee71d031cc2ad72a6de1da133946f8681f166f180c5a7b7c8bace401fR25? Or Musca S1 is not affected?

Reply:
MUSCA S! already has it... this was done as part of https://github.com/ARMmbed/mbed-os/pull/13196/commits/37f26692b1db388a272c9037b28cad27734fda90